dot/types/account.go

```diff
@@ -21,3 +21,11 @@ type AccountInfo struct {
 		FreeFrozen *scale.Uint128
 	}
 }
+
+// AccountData represents account data of an account. Currently is used in core tests
+type AccountData struct {
```

Comment:
Can you use this type instead of the anonymous struct in the Account struct in the `types` package as well?
And then update the comment as well for this struct.

Reply:
yup good call!